third_party/move/move-compiler-v2/tests/checking/typing/expected_actual_assign.exp

```diff
@@ -0,0 +1,31 @@
+
+Diagnostics:
+error: cannot assign `u8` to left-hand side of type `u64`
+   ┌─ tests/checking/typing/expected_actual_assign.move:13:9
+   │
+13 │         x = 1u8;
```

Comment:
Hmm. This is happening in 100 places in our baseline files, and not introduced in this PR, so this is just a new test case showing it to you. I personally find the error message pretty clear. We can come back to this at a later point, but we have been living with this error message since a long time, and I feel reluctant to open a bug for this. We have generally on the agenda to perhaps review error messages in the future.

Reply:
I added a feature request (12733), so we don't have to go hunting again.